staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -176,6 +178,9 @@ func (s *SecureServingOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.ServerCert.CertKey.KeyFile, "tls-private-key-file", s.ServerCert.CertKey.KeyFile,
 		"File containing the default x509 private key matching --tls-cert-file.")
 
+	fs.StringVar(&s.ServerCert.CertKey.PeerCAFile, "peer-ca-file", s.ServerCert.CertKey.PeerCAFile,
+		"File containing trust bundle used to verify serving certificates for other kube-apiserver instances")
```

Comment:
This code location is part of the generic API server library, which kube-apiserver is just one instance of.

Are all API servers being given this peer-proxying capability, or just kube-apiserver / kube-aggregator?

If so, this help should be more generic instead of mentioning `kube-apiserver` by name.

If not, I expected this to live in `AggregatorOptions#AddFlags` or `ServerRunOptions#Flags` to be scoped to kube-apiserver

Reply:
Ah yes, thanks for the hint. The new flag is meant for just kube-apiserver. Moved the changes over to
- `ExtraConfig in cmd/kube-apiserver/app/aggregator.go`
- `ServerRunOptions in cmd/kube-apiserver/app/options/options.go`
- `AggregatorOptions in staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go`

This looks so much cleaner.